lib/hanami/config/logger.rb

```diff
@@ -113,27 +119,18 @@ class Logger
       # @api private
       def initialize(env:, app_name:)
         @app_name = app_name
-
-        config.level = case env
-                       when :production
-                         :info
-                       else
-                         :debug
-                       end
-
-        config.stream = case env
-                        when :test
-                          File.join("log", "#{env}.log")
-                        else
-                          $stdout
-                        end
-
-        config.formatter = case env
-                           when :production
-                             :json
-                           else
-                             :rack
-                           end
+        @env = env
+
+        case env
+        when :development, :test
```

Comment:
@solnic Shouldn't development be stdout?

Reply:
@jodosha $stdout is the default stream, so no need to set it again here